src/pages/Settings/Settings.vue

```diff
@@ -283,7 +313,10 @@ export default defineComponent({
     async onValueChange(field: Field, value: DocValue): Promise<void> {
       const { fieldname } = field;
       delete this.errors[fieldname];
-
+      if (fieldname === 'updateButton') {
+        await this.handleManualUpdateCheck();
+        return;
+      }
       try {
         await this.doc?.set(fieldname, value ?? '');
```

Comment:
Fallback to empty string will open doors for new runtime bugs here. 

We have fields of mixed types in the Settings Page right ?. When this logic tries to set empty string value to a `boolean` it will fail.  Please revert this and see if you could find alternate solutions.